python/taichi/misc/image.py

```diff
@@ -16,15 +16,37 @@ def imwrite(img, filename):
     assert len(img.shape) in [2,
                               3], "Image must be either RGB/RGBA or greyscale"
 
-    resx, resy = img.shape[:2]
     if len(img.shape) == 2:
-        comp = 1
+        img = img.reshape(*img.shape, 1)
+
+    assert img.shape[2] in [1, 3, 4], "Image must be either RGB/RGBA or greyscale"
+
+    return img.swapaxes(0, 1)[::-1, :]
+
+
+
+def imdisplay(img):
+    """
+    Try to display image in interactive shell.
+    """
+    if ti.lang.shell.oinspect.name.startswith('IPython'):
```

Comment:
Does this condition evaluate to `True` in command line IPython? We probably only want `IPython.display` for Jupyter Notebook.